job_creator.php

```diff
@@ -213,6 +210,16 @@ private function getPhpVersion(int $phpIndex): string
                 $key = sprintf('%d.%d', $cmsMajor, $matches[1]);
             }
         }
+
+        return $key;
```

Comment:
Rename this variable in `getBranchVersion()` from `$key` to `$version` since that's what it represents in the context of this method.

Reply:
Also, if "branch version" is actually just the name of the branch, please rename the method to `getBranchName()`. If it's the version of installer which will be installed, I think there's a method for that already? And if it's something else, please add a PHPDoc to explain what the return value represents.

Reply:
Updated